rebasehelper/tests/functional/test_rebase.py

```diff
@@ -111,3 +112,57 @@ def test_rebase(self, buildtool):
             assert lib['Variables changes summary']['Removed']['count'] == 1
         repo = git.Repo(execution_dir)
         assert '- New upstream release {}'.format(self.NEW_VERSION) in repo.commit().summary
+
+    @pytest.mark.parametrize('buildtool', [
+        pytest.mark.skipif(
+            os.geteuid() != 0,
+            reason='requires superuser privileges')
+        ('rpmbuild'),
+        pytest.mark.long_running('mock'),
+    ])
+    @pytest.mark.parametrize('favor_on_conflict', ['upstream', 'downstream'])
```

Comment:
Actually what I had in mind was to parametrize `test_rebase` method itself, with `favor_on_conflict` being `None`, `'upstream'` and `'downstream'`.